.npmrc

```diff
@@ -1,7 +1,11 @@
 # Don't create or read package-lock.json
 package-lock=false
+
 # Use exact versions of dependencies vs. semver minor updates
 save-exact=true
 
 # Allow running postinstall script in Docker
 unsafe-perm=true
+
+# Allows users of npm7.x to properly build Telescope without breaking. Uses old npm6.x peer dependency installation
+legacy-peer-deps=true
```

Comment:
Add a newline after this

Reply:
Will do! I'll also rebase to master so that we can clean things up a bit.